app/views/character_types/_form.html.erb

```diff
@@ -1,14 +1,14 @@
-<%= form_with(model: character_type) do |form| %>
-  <% if character_type.errors.any? %>
-    <div id="error_explanation">
-      <h2><%= pluralize(character_type.errors.count, "error") %> prohibited this character_type from being saved:</h2>
-      <ul>
-        <% character_type.errors.full_messages.each do |message| %>
-          <li><%= message %></li>
-        <% end %>
-      </ul>
-    </div>
-  <% end %>
+<%= form_with(model: character_type, local:true) do |form| %>
+  <%# if character_type.errors.any? %>
```

Comment:
@StefanGajic not sure why you commented out these.

Reply:
mistake, I forgot to uncomment them